rpg.py

```diff
@@ -2,29 +2,42 @@
 from queue import PriorityQueue
 from typing import List, Tuple
 import random
+import time
+
+
+class Item:
+    def __init__(self,
+                 name: str,
+                 hp_restore: int):
+        self.name = name
+        self.hp_restore = hp_restore
+
+
+inventory = []
```

Comment:
Shouldn't inventory be per-character?

Reply:
there's just one inventory and all the ally characters can access it